### PR TITLE
DDF-2428 Add docs for Resource Management apps

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Adds documentation for the Resource Management application to the documentation. 

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@bdeining @jlcsmith @Lambeaux @jrnorth @mcalcote 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@pklinef

#### How should this be tested?

review outline, builds successfully

#### Any background context you want to provide?

Added to `draft-documentation` to be included in documentation rewrite. For info on rewrite, see https://github.com/codice/ddf/pull/1036.
#### What are the relevant tickets?

https://codice.atlassian.net/browse/DDF-2428
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+The ${ddf-resource-management} Application provides administrative functionality to impose data usage limits on users, as well as the ability to view and terminate searches that are in progress.
```
Comment (jlcsmith) [inline distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc]: probably should add a note here that it allows an admin to view data usage as well as impose limits.  Perhaps a list of capabilities is appropriate here.. view data usage, impose data limits, view and cancel running searches...

File: distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+|None
+|
+|
+
+|===
+
+=== ${ddf-resource-management} Data Usage Tab
+
+Configure users' data limits and reset times for those limits.
```
Comment (jlcsmith) [inline distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc]: add "view data usage and ..."

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc]: added.

File: distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc]: Added those three. Are there more?

File: distribution/docs/src/main/resources/documentation.adoc [hunk 3]
```diff
@@ -77,6 +77,13 @@ include::${project.build.directory}/doc-contents/_contents/_registry-contents/ma
 
 <<<
 
+== Managing ${ddf-resource-managemen}
```
Comment (jrnorth) [inline distribution/docs/src/main/resources/documentation.adoc]: `managemen`

File: distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc [hunk 4]
```diff
@@ -0,0 +1,53 @@
+
+The ${ddf-resource-management} Application provides administrative functionality to monitor and manage data usage on the system.
+This application allows an administrator to:
+
+* View data usage
+* set limits on users
+* view and terminate searches that are in progress.
```
Comment (bdeining) [inline distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc]: nit : capitalize `set` and `view`

File: distribution/docs/src/main/resources/documentation.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/documentation.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc (hunk 4 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc]: fixed.

**Comment by jlcsmith** [comment]
Built fine.  LGTM 👍

**Comment by bdeining** [comment]
👍

**Comment by jlcsmith** [comment]
@ricklarsen - needs to be rebased please.

**Comment by ricklarsen** [comment]
@jlcsmith Will do, but it is waiting on the ones before it to avoid merge conflicts.

**Comment by shaundmorris** [comment]
@ricklarsen can you please resolve the conflicts on this one?

**Comment by ricklarsen** [comment]
Conflicts resolved.

**Comment by shaundmorris** [comment]
Build was successful on Bamboo

**Pull request merged**